api/src/notifications/models.py

```diff
@@ -27,6 +28,16 @@ class Notification(models.Model):
     viewers = models.ManyToManyField(
         to=CustomUser, related_name="viewed_notifications", blank=True
     )
+    notification_type = models.CharField(
+        choices=settings.NOTIFICATION_CHOICES,
+        default=settings.LIKES_FIELD,
+        max_length=max(
+            [
+                len(notification_type[0])
+                for notification_type in settings.NOTIFICATION_CHOICES
+            ]
+        ),
+    )
```

Comment:
In other parts of the code, we use the `models.PositiveSmallIntegerField` for choice fields, and pass in a list of tuples of (int, string). We probably want to do the same.